### PR TITLE
Improve the categories styling

### LINKED ISSUE #12: Revise display of selected categories/channels
After looking at the categories/channels selections, I feel that the better solution would have them be colored when selected (colored label) and uncolored when not. Much better than clicking a little eye :).

### PR DESCRIPTION
**Pull request by Flibio**
Improves the categories styling. Closes #12. Changes include:
- Made the entire row of a category clickable
- Remove the "eye" icon
- Made the background color of the category change to a light gray when selected

### CODE REVIEW

File: app/views/projects/table.scala.html [hunk 1]
```diff
@@ -73,33 +73,28 @@ <h3 class="panel-title">@messages("project.category.plural")</h3>
...
                     }
                 </div>
-                <table class="table panel-body">
+                <table class="table panel-body category-table">
                     <tbody>
                         @for(i <- 0 until Categories.values.size) {
-                            <tr>
+                            @if(visibleCategories.isDefined) {
+                                @if(!visibleCategories.get.contains(Categories(i).asInstanceOf[Category])) {
+                                    <tr onclick="location.href='@routes.Application.index(Some(
```
Comment (windy1) [inline app/views/projects/table.scala.html]: What's your reasoning for using js here?

Comment (windy1) [inline app/views/projects/table.scala.html]: Oh bc its not an anchor....duh

Comment (Flibio) [inline app/views/projects/table.scala.html]: Yeah it is the only way I could think of to make the entire row a link without making multiple anchors.

**Comment by Flibio** [comment]
@windy1 Is there anything I need to change on this PR?

**Comment by windy1** [comment]
Hmm I'm honestly not crazy about the glow effect you got going on there it looks kind of odd.

**Comment by Flibio** [comment]
I thought that it looked kind of weird as well. What did you mean when you said that you wanted the text to glow?

**Comment by windy1** [comment]
For reference:

Screenshot

This kind of looks like it has a "cloud" around the text rather than glowing (how can something black emit yellow light). I was thinking something more along the lines of the actual text glowing not too unlike this: http://enjoycss.com/gallery/text_effects/39

Feel free to take another route though as I'm definitely open to suggestions: glowing text was just one.

**Comment by Flibio** [comment]
Alright, I see what you mean. I'll try to improve the look of it.

**Comment by Zidane** [comment]
Actually this isn't what I meant but I can't properly relay it as I'm not a graphics artist. I'll have Grinch make up a mockup so that you guys know what I want.

**Comment by Flibio** [comment]
@Zidane Ok, just post it here when it is completed.

**Comment by ryantheleach** [comment]
I assume zidane wants the background color to a slightly brighter value, and the text color from black to colored.

**Comment by Grinch** [comment]
This is a generic mockup put together of what is desired. Basically, if nothing is 'highlighted' (for the lack of better words) we default to showing all categories. If one or more items are highlighted we only show those items.

Example

**Note:** I'm not sure the color in my example would be the best choice though.

**Comment by Flibio** [comment]
@Grinch I think a yellow color would be a bit much once you start selecting a few categories, so the gray you have in the mockup will probably be the best color.

**Comment by windy1** [comment]
I still like glowing things :P 

How about a colored inner shadow for the row where the border-top and border-bottom for each consecutive row are broken to make it look like it's multi-selected

Eg:

```
---------------------
| Selected          |
| Selected          |
| Selected          |
---------------------
| Not selected      |
---------------------
| Selected          |
---------------------
```

**Comment by Grinch** [comment]
@Flibio I agree.

@windy1 It's cool yes, but the design our websites follow does not fit it very well in my opinion.

**Comment by windy1** [comment]
Well I'm moving off from glowing text now and just suggesting a colored inner-border shadow for selected categories instead of highlighting the whole row.

**Comment by Grinch** [comment]
@windy1 Could you show an example please?

**Comment by windy1** [comment]
@Grinch 

categories

Ignore the random color and invalid measurements, I just did this quickly in chrome :p but I think it gives you the general idea.

The bottom and/or top border would be removed for consecutively selected categories.

**Comment by Flibio** [comment]
@Grinch 

What do you think of that style suggested by windy?

**Comment by Grinch** [comment]
@Flibio @windy1 I'm not against it. I propose we try it and if it doesn't fit/work well we can try my suggestion.

**Comment by Flibio** [comment]
@windy1 I've added the glow effect. I'm not very good with Scala, so I have not added the part where the bottom and/or top border would be removed for consecutively selected categories. Hopefully that is okay.

**Comment by windy1** [comment]
Nice work 👍

**Comment by Zidane** [comment]
categories highlight

Not feeling the way this highlight seems. It doesn't stand out. Still thinking that the whole background highlight that Grinch had above is the ticket.

**Comment by ryantheleach** [comment]
or both, but make the glow/shadow more subtle.

**Comment by simon816** [comment]
screenshot
Just added `background: #F6d830;` to the `.category-table div.glow` selector
(a lighter variant of yellow)

**Comment by windy1** [comment]
I rather like that @simon816

**Comment by Zidane** [comment]
@windy1 

Basically what @simon816 did is what I was looking for. Apologies if I wasn't relaying that correctly.

This should be applied to any category like things in Ore (including release channels).

**Pull request merged**